process/dnsProcessor.go

```diff
@@ -15,12 +15,16 @@ import (
 
 var usernameHasher = &keccak.Keccak{}
 
+const usernameHashLength = 32
+
 const (
 	lowUsernameLengthBoundary  = 3
-	highUsernameLengthBoundary = 20
+	highUsernameLengthBoundary = 25
 	usernameSuffix             = ".elrond"
 )
 
+var isUsernameAlphanumeric = regexp.MustCompile(`^[a-z0-9_]*$`).MatchString
```

Comment:
regex are pretty slow. Let's refactor and have the performance as high as we can get.

Reply:
agree. run some benchmarks and the difference was 40x. refactored